canonical-web-and-design/anbox-cloud.io
# Create game-streaming vertical page

## Done

- Create game-streaming vertical page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/gaming
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [Copy-doc](https://docs.google.com/document/d/1c9WcMRx1OlQxUntC6YF5oq_wjmwA5vI-RDZC926EYmo/edit#)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4009
